mm2src/mm2_metrics/src/recorder.rs

```diff
@@ -154,20 +153,16 @@ impl MmRecorder {
     }
 }
 
+use std::collections::hash_map::Entry::{Occupied, Vacant};
```

Comment:
It's better to put imports at the beginning of the file

Reply:
done